package.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Again, there are a bunch of old changes that are already on the 'master' branch showing as 'new' changes in this and the following files. Not sure why this is happening.